jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1262,6 +1271,13 @@ public boolean levelChanged(@NotNull AbstractEndpoint endpoint, long level)
         SpeakerRanking ranking = speechActivity.levelChanged(endpoint, level);
         if (ranking == null || !routeLoudestOnly)
             return false;
+        // return false if the source is explicitly subscribed by any other endpoint.
+        List<AudioSourceDesc> sources = endpoint.getAudioSources();
+        for (AudioSourceDesc source : sources) {
```

Comment:
This is one place that we want to optimize, as it runs very often -- 50 times per second per audio source, even when muted. The existing case (no endpoint using the new signaling) should not degrade.